### ISSUE TITLE
Outdated Client: workflow/v1alpha1

### ISSUE THREAD

**Issue by serafdev** [argoproj/argo-workflows#8077]
## Summary

Can't compile Go code that uses the `v1alpha1.WorkflowInterface`.

## Diagnostics

Trying to run the following:

```go
wfClient := wfclientset.NewForConfigOrDie(conf).ArgoprojV1alpha1().Workflows(namespace)
wfClient.Create(
    ctx,
    workflow(opts), // Wrapper to a *wfv1.Workflow
    metav1.CreateOptions{},
)
```

Error I'm getting:

```sh
❯ go vet ./...
# github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:139:3: cannot use "github.com/go-openapi/spec".Schema{...} (type "github.com/go-openapi/spec".Schema) as type "k8s.io/kube-openapi/pkg/validation/spec".Schema in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:151:3: cannot use "github.com/go-openapi/spec".Schema{...} (type "github.com/go-openapi/spec".Schema) as type "k8s.io/kube-openapi/pkg/validation/spec".Schema in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:158:8: cannot use ref("github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1.Tar...) (type "k8s.io/kube-openapi/pkg/validation/spec".Ref) as type "github.com/go-openapi/spec".Ref in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:163:8: cannot use ref("github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1.Non...) (type "k8s.io/kube-openapi/pkg/validation/spec".Ref) as type "github.com/go-openapi/spec".Ref in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:168:8: cannot use ref("github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1.Zip...) (type "k8s.io/kube-openapi/pkg/validation/spec".Ref) as type "github.com/go-openapi/spec".Ref in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:181:3: cannot use "github.com/go-openapi/spec".Schema{...} (type "github.com/go-openapi/spec".Schema) as type "k8s.io/kube-openapi/pkg/validation/spec".Schema in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:200:11: cannot use ref("github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1.Par...) (type "k8s.io/kube-openapi/pkg/validation/spec".Ref) as type "github.com/go-openapi/spec".Ref in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:220:11: cannot use ref("github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1.Art...) (type "k8s.io/kube-openapi/pkg/validation/spec".Ref) as type "github.com/go-openapi/spec".Ref in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:280:8: cannot use ref("github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1.S3A...) (type "k8s.io/kube-openapi/pkg/validation/spec".Ref) as type "github.com/go-openapi/spec".Ref in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:286:8: cannot use ref("github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1.Git...) (type "k8s.io/kube-openapi/pkg/validation/spec".Ref) as type "github.com/go-openapi/spec".Ref in field value
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.2.9/pkg/apis/workflow/v1alpha1/openapi_generated.go:286:8: too many errors
```

---

**Message from the maintainers**:

Impacted by this regression? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by serafdev** [comment]
Seems to be fixed in `v3.3.0-rc8` (maybe slightly before, rc1 is broken)

**Comment by alexec** [comment]
This was fixed in a recent PR. That can’t be backported to v3.2, because is upgrade client-go. Please use v3.3
